<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerSettingsWindow</class>
 <widget class="QWidget" name="ControllerSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DuckStation Controller Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/duckstation-qt.qrc">
    <normaloff>:/icons/duck.png</normaloff>:/icons/duck.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="settingsCategory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="settingsContainer">
     <property name="minimumSize">
      <size>
       <width>1100</width>
       <height>620</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="controllerProfileLayout">
     <item>
      <layout class="QHBoxLayout" name="editProfileLayout">
       <item>
        <widget class="QLabel" name="editProfileLabel">
         <property name="text">
          <string>Editing Preset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="currentProfile">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newProfile">
         <property name="text">
          <string>New Preset</string>
         </property>
         <property name="icon">
          <iconset theme="add-line"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyProfile">
         <property name="text">
          <string>Apply Preset</string>
         </property>
         <property name="icon">
          <iconset theme="folder-open-line"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteProfile">
         <property name="text">
          <string>Delete Preset</string>
         </property>
         <property name="icon">
          <iconset theme="minus-line"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copyGlobalSettings">
         <property name="text">
          <string>Copy Global Settings</string>
         </property>
         <property name="icon">
          <iconset theme="folder-open-line"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Close|QDialogButtonBox::StandardButton::RestoreDefaults</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/duckstation-qt.qrc"/>
 </resources>
 <connections/>
</ui>
